langchain4j-vertex-ai-gemini/src/main/java/dev/langchain4j/model/vertexai/RoleMapper.java

```diff
@@ -0,0 +1,17 @@
+package dev.langchain4j.model.vertexai;
+
+import dev.langchain4j.data.message.ChatMessageType;
+
+class RoleMapper {
+
+    static String map(ChatMessageType type) {
+        switch (type) {
+            case SYSTEM:
```

Comment:
According to Gemini API: https://cloud.google.com/vertex-ai/docs/generative-ai/model-reference/gemini
USER: Specifies content that's sent by you.
MODEL: Specifies the model's response.

Maybe SYSTEM should map to "model"?

Reply:
We have agreed with @glaforge to deny system messages alltogether for the moment as they are not supported by Gemini (yet?)

Reply:
As @langchain4j says, we think it's better to disallow system messages as they are not supported by Gemini.
Furthermore, if you pass a "model" message as a first message, then follow up with "user"'s messages, it'll complain anyway that it's not the right user/model alternation of messages.
So it has to be user/model/user/model/etc.
If you want to mimic system messages, the first user message should start with the system context: "You're an expert chatbot in blahblah/n/nHow do I do this?"